### ISSUE TITLE
Server crashed on zombie horse spawn / zombie horse lightning strike??

### ISSUE THREAD

**Issue by kinnikinnick** [Faboslav/variants-and-ventures#33]
### Minecraft version information

1.20.1

### Mod loader information

Forge

### Mod loader version information

47.3

### Mod version information

1.0.6

### Expected Behavior

The server should not crash

### Actual Behavior

The server crashes

### Reproduction Steps

I'm not sure..  I could try spawning the mob I guess?

### Screenshots & files

[crash-2024-12-25_20.29.53-server.txt](https://github.com/user-attachments/files/18250830/crash-2024-12-25_20.29.53-server.txt)

**Comment by Faboslav** [comment]
I tried to look into this, but could not find anything :/.

**Comment by Faboslav** [comment]
Yeah, without the reproduction step i cant really get anything to crash on my own, if you find out the way to get the crash, please let me know :).
